src/CommunityToolkit.Maui/Options.cs

```diff
@@ -1,32 +1,83 @@
 using CommunityToolkit.Maui.Behaviors;
 using CommunityToolkit.Maui.Converters;
+using Microsoft.Maui.LifecycleEvents;
 
 namespace CommunityToolkit.Maui;
 
 /// <summary>
 /// .NET MAUI Community Toolkit Options.
 /// </summary>
-public class Options : Core.Options
+public class Options() : Core.Options
 {
+	readonly MauiAppBuilder? builder;
+
+	internal Options(MauiAppBuilder builder) : this()
+	{
+		this.builder = builder;
+	}
+
 	internal static bool ShouldSuppressExceptionsInAnimations { get; private set; }
 	internal static bool ShouldSuppressExceptionsInConverters { get; private set; }
 	internal static bool ShouldSuppressExceptionsInBehaviors { get; private set; }
+	internal static bool ShouldEnableSnackbarOnWindows { get; private set; }
 
 	/// <summary>
 	/// Allows to return default value instead of throwing an exception when using <see cref="BaseConverter{TFrom,TTo}"/>.
-	/// Default value is false.
 	/// </summary>
+	/// <remarks>
+	/// Default value is false.
+	/// </remarks>
 	public void SetShouldSuppressExceptionsInConverters(bool value) => ShouldSuppressExceptionsInConverters = value;
 
 	/// <summary>
 	/// Allows to return default value instead of throwing an exception when using <see cref="AnimationBehavior"/>.
-	/// Default value is false.
 	/// </summary>
+	/// <remarks>
+	/// Default value is false.
+	/// </remarks>
 	public void SetShouldSuppressExceptionsInAnimations(bool value) => ShouldSuppressExceptionsInAnimations = value;
 
 	/// <summary>
 	/// Allows to return default value instead of throwing an exception when using <see cref="BaseBehavior{TView}"/>.
-	/// Default value is false.
 	/// </summary>
+	/// <remarks>
+	/// Default value is false.
+	/// </remarks>
 	public void SetShouldSuppressExceptionsInBehaviors(bool value) => ShouldSuppressExceptionsInBehaviors = value;
+
+	/// <summary>
+	/// Enables <see cref="Alerts.Snackbar"/> for Windows
+	/// </summary>
+	/// <remarks>
+	/// Additional setup is required in the Package.appxmanifest file to enable Snackbar on Windows. See the <see cref="Alerts.Snackbar"/> <a href="https://learn.microsoft.com/dotnet/communitytoolkit/maui/alerts/snackbar">Platform Specific Initialization Documentation</a> for more information. Default value is false.
+	/// </remarks>
+	public void SetShouldEnableSnackbarOnWindows(bool value)
+	{
```

Comment:
If value is false return from the method

Reply:
Or just change to `public void EnableSnackbarOnWindows()`?

Reply:
I think it'd still be smart for us to set `Options.ShouldEnableSnackbarOnWindows = value` to match the existing implementations like `ShouldSuppressExceptionsInBehaviors`. We may want to use it for something in the future (or not, but it's there if we need it!).

I just added a bit of control flow to check if `value is true && builder is not null`, and then throw an `InvalidOperationException` if for some reason a user tries to set this value outside of `.UseMauiCommunityToolkit()`.



```cs
#if WINDOWS
		if (value is true && builder is null)
		{
			throw new InvalidOperationException($"{nameof(SetShouldEnableSnackbarOnWindows)} must be called using the {nameof(AppBuilderExtensions.UseMauiCommunityToolkit)} extension method. See the Platform Specific Initialization section of the {nameof(Alerts.Snackbar)} documentaion for more inforamtion: https://learn.microsoft.com/dotnet/communitytoolkit/maui/alerts/snackbar)");
		}
		else if (value is true && builder is not null)
		{
			builder.ConfigureLifecycleEvents(events =>
			{
				events.AddWindows(windows => windows
					.OnLaunched((_, _) =>
					{
						Microsoft.Windows.AppNotifications.AppNotificationManager.Default.NotificationInvoked += OnSnackbarNotificationInvoked;
						Microsoft.Windows.AppNotifications.AppNotificationManager.Default.Register();
					})
					.OnClosed((_, _) =>
					{
						Microsoft.Windows.AppNotifications.AppNotificationManager.Default.NotificationInvoked -= OnSnackbarNotificationInvoked;
						Microsoft.Windows.AppNotifications.AppNotificationManager.Default.Unregister();
					}));
			});

			static void OnSnackbarNotificationInvoked(Microsoft.Windows.AppNotifications.AppNotificationManager sender,
														Microsoft.Windows.AppNotifications.AppNotificationActivatedEventArgs args)
			{
				Alerts.Snackbar.HandleSnackbarAction(args);
			}
		}
#endif
```

Reply:
Ah - I figured out the perfect use-case for `Options.ShouldEnableSnackbarOnWindows`! 

We can use it verify the constructor of 

```cs
public Snackbar()
	{
		if (OperatingSystem.IsWindows() && !Options.ShouldEnableSnackbarOnWindows)
		{
			throw new InvalidOperationException($"Additional setup is required in the Package.appxmanifest file to enable {nameof(Snackbar)} on Windows. Additonally, `{nameof(AppBuilderExtensions.UseMauiCommunityToolkit)}(options => options.{nameof(Options.SetShouldEnableSnackbarOnWindows)}({true.ToString().ToLower()});` must be called to enable Snackbar on Windows. Additonally, Platform Specific Initialization section of the {nameof(Snackbar)} documentaion for more information: https://learn.microsoft.com/dotnet/communitytoolkit/maui/alerts/snackbar")
			{
				HelpLink = "https://learn.microsoft.com/dotnet/communitytoolkit/maui/alerts/snackbar"
			};
		}
		// ...
	}
```

Reply:
^This'll also help every dev who inevitably doesn't read the docs (like me 😀) to be alerted to the new breaking change requirement on Windows for Snackbar.